core/src/main/java/com/datastrato/gravitino/storage/relational/mapper/CatalogMetaMapper.java

```diff
@@ -149,14 +149,14 @@ Integer updateCatalogMeta(
   @Update(
       "UPDATE "
           + TABLE_NAME
-          + " SET deleted_at = UNIX_TIMESTAMP(CURRENT_TIMESTAMP(3)) * 1000.0"
+          + " SET deleted_at = (UNIX_TIMESTAMP() * 1000.0) + EXTRACT(MICROSECOND FROM CURRENT_TIMESTAMP(3)) / 1000"
```

Comment:
I  guess it is too long to exceed 100 chars, please check and fix them.

Reply:
done